packages/auth/src/auth-options.ts

```diff
@@ -84,7 +84,7 @@ export const getAuthOptions = ({
           return session
         }
 
-        await core.auth0SynchronizationService.synchronizeUser(user)
+        await core.auth0SynchronizationService.handleUserSync(user)
```

Comment:
I think this makes more sense, as it most often does not do any synchronization?

You can debate it though, because from the perspective of the contract, the method just makes sure that the user is synchronized between auth0 and the database.